### ISSUE TITLE
How can I hint the web port to kitematic?

### ISSUE THREAD

**Issue by withinboredom** [kitematic/kitematic#493]
Creating a docker file where the software has an unconfigurable web port (consul, in case you're wondering). How can I hint to kitematic that the http port is on a strange one?

**Comment by mchiang0610** [comment]
@withinboredom Are you referring to specifying to Kitematic that your web port needs a web preview? 

If so, that feature is not available yet. Will change this into a feature request. Thanks!

**Comment by withinboredom** [comment]
Yes, the web port is on 8500 or some such. Even if it were just a global list of ports to try displaying a web preview. I don't think it needs to be per image, IMO.

**Comment by mchiang0610** [comment]
@withinboredom Thanks for the clarification.

**Comment by mchiang0610** [comment]
+1 to editable port mappings from @johncokos

**Comment by scytacki** [comment]
I just found the bit of code that determines if Kitematic opens the webPreview or not. These might be useful if anyone starts working on this.  https://github.com/docker/kitematic/blob/e40ba41f1b2d623636241162df0b44178a685af6/src/utils/Util.js#L215

**Comment by scytacki** [comment]
I just found the bits of code that determines if Kitematic opens the webPreview or not. These might be useful if anyone starts working on this.
Kitematic looks for the first port on the container that is a 'webPort':
https://github.com/docker/kitematic/blob/e40ba41f1b2d623636241162df0b44178a685af6/src/components/ContainerDetails.react.js#L20
This list of webPorts:
https://github.com/docker/kitematic/blob/e40ba41f1b2d623636241162df0b44178a685af6/src/utils/Util.js#L215
Perhaps the list of webPorts was configurable it would address this feature request.
